includes/polling/os/timos.inc.php

```diff
@@ -10,13 +10,16 @@
 
 $version = 'v' . $majorVersion . '.' . $minorVersion . '.' . $versionModifier;
 
-//SNMPv2-MIB::sysDescr.0 = STRING: TiMOS-B-9.0.R3 both/hops Nokia SAS-Sx 48Tp4SFP+ (PoE) 7210 Copyright (c) 2000-2017 Nokia.
-//All rights reserved. All use subject to applicable license agreements.
-//Built on Thu Jan 5 11:01:16 IST 2017 by builder in /home/builder/9.0B1/R3/panos/main
+$chassis_type_name_array = snmpwalk_cache_oid($device, 'tmnxChassisTypeName', $a = array(), 'TIMETRA-CHASSIS-MIB', 'aos', '-OQUs');
+$hardware = end(reset($chassis_type_name_array));
```

Comment:
To fetch the actual chassis model and type. without resorting to a regex of the sysDescr (which wasn't working on v9 of SROS, but was on v8)  

I almost used `tmnxChassisTypeDescription` to populate `$features` but decided against it.  

On a SAS-K the walk returns two nodes:

```
snmpwalk -M /usr/share/snmp/mibs/:Code/public_projects/librenms/mibs:Code/SNMP-mibs/nokia -m +TIMETRA-PORT-MIB -v2c -c public -OQun 10.1.86.31 tmnxChassisTypeName
.1.3.6.1.4.1.6527.3.1.2.2.1.6.1.2.20 = "7210 SAS-K 2F2T1C-1"
.1.3.6.1.4.1.6527.3.1.2.2.1.6.1.2.36 = "7210 VC"
```

If they ever get our SAS-T's powered, I'll test against those too; but I don't expect them to  return a lot of additional stuff, based on the documentation in the MIB.

The .20 and .36 endpoints are not readily identifiable through another direct `get` that I can find, so a narrow walk and reading the first object in the array seemed like the best past forward.